### ISSUE TITLE
333

### ISSUE THREAD

**Issue by net968** [MetaMask/eth-phishing-detect#2377]
333 good project

**Comment by net968** [comment]
The website 333eth.io is not a Scam it is a mistake thank you

**Comment by votesher36** [comment]
333ETH.IO - IS NOT A SCAM AND NOT A FISHING SITE.
This is the machinations of competitors. Unlock this site, please

**Comment by human228** [comment]
Not a scam

**Comment by 409H** [comment]
Closing. Duplicate of https://github.com/MetaMask/eth-phishing-detect/issues/2374

**Comment by admin333eth** [comment]
Hello,
I am the lead admin of 333eth.io. As you may know, our project was launched on September 4 and has been a huge success so far. The key advantage of 333ETH is that it is fully autonomous and functions on the smart contract only, without our interference. The smart contract was developed from scratch by our team.
This is not our first project: our team is also behind https://ponzitrust.com/en/ - a project that opened 7 months ago and has proved its fairness and honesty. In fact, the largest contributors to 333ETH are PonziTrust participants – they trust us and keep asking us not to publish the code!
Our project’s success has attracted lots of attention, which is not surprising given how much money is being contributed to it. Quite a few competitors have emerged, using all possible methods (including blackmail and threats) to try and make us publish the code that they could simply copy and launch identical projects without doing any work. What we are worried about, however, is not that similar projects will appear, but rather that hackers will get their hands on the code and try to use it to hack the contract. We cannot risk our members’ trust this way – after all, they all support the idea of not publishing the code and doing an independent smart contract audit instead. To all new contributors, we offer an opportunity to review the smart contract code personally in our office. What’s more, an audit of our contract by the well-known company https://www.ххххххх.com/(I will only report to admin gitHUB) has already begun and will be finished in a week! We could not do this earlier due to the hefty price tag for the audit - $20 000, which we simply did not have when the project started. 
If you review how this unpleasant situation unfolded, you’ll see that it was the work of just one user who created his or her account specifically for this purpose yesterday – on September 11. Do you think it’s a coincidence? Surely not – it’s not the first such attack on us, and we are used to it.
I kindly ask you to review the possibility of unblocking the site of 333ETH until the audit by https://www.ххххххх.com/(I will only report to admin gitHUB) is completed. Cannot you see that you are being manipulated? Is it reasonable that just one person – acting without any evidence – should be able to get our project blocked?
I will be pleased to answer any questions and provide additional information
